biz.aQute.launcher/src/aQute/launcher/plugin/ProjectLauncherImpl.java

```diff
@@ -55,7 +55,9 @@
 public class ProjectLauncherImpl extends ProjectLauncher {
 	private final static Logger		logger					= LoggerFactory.getLogger(ProjectLauncherImpl.class);
 	private static final String		EMBEDDED_LAUNCHER_FQN	= "aQute.launcher.pre.EmbeddedLauncher";
-	private static final String		EMBEDDED_LAUNCHER		= "aQute/launcher/pre/EmbeddedLauncher.class";
+	private static final String[]	EMBEDDED_CLASSES		= new String[] {
+		"aQute/launcher/pre/EmbeddedLauncher.class", "aQute/libg/classloaders/ModifiableURLClassLoader.class"
```

Comment:
This pulls in the the complete `aQute/launcher/pre` package which is part of the launcher project. It now pulls in part of the `aQute/libg/classloaders` package from another project. Bit worried about the partial package thing...

Reply:
Not sure how else to solve this.

Reply:
Well the new Modifiable class loader could be part of the pre package in the launcher since it is a special-for-bnd-launcher class loader, no?

Reply:
It's also used in bndlib for launching in other similar scenarios.

Reply:
But you made that change in this PR and I would submit it is wrong to change the expected parent-first semantics there.